pkgs/development/libraries/libpointmatcher/default.nix

```diff
@@ -0,0 +1,41 @@
+{stdenv, fetchFromGitHub, cmake, eigen, boost, libnabo}:
+
+let inherit (stdenv.lib) getVersion versionAtLeast; in
+
+assert versionAtLeast (getVersion eigen) "3";
```

Comment:
You mean all three of them? Ok, I will remove these then.

Just to clearify: If a package depends on a specific version, I would write the expression with say, eigen2 as a function parameter instead?